### PR TITLE
[Feature] Strikethrough Support

### PR DESCRIPTION
**Pull request by jgsamudio**
Updated TextStyle to handle strike through on text.

### CODE REVIEW

File: Marker/Classes/TextStyle.swift [hunk 1]
```diff
@@ -157,7 +160,20 @@ public struct TextStyle {
         self.paragraphSpacingBefore = paragraphSpacingBefore
         self.textAlignment = textAlignment
         self.lineBreakMode = lineBreakMode
+        self.strikeThroughStyle = strikeThroughStyle
         self.textTransform = textTransform
     }
-    
 }
+//
```
Comment (htinlinn) [inline Marker/Classes/TextStyle.swift]: Get rid of this ;)

File: Marker/Classes/Markdown/EmphasisTag.swift [hunk 2]
```diff
@@ -20,13 +20,15 @@ internal enum EmphasisTag {
     case underscoreEm(Index)
     case asteriskStrong(Index)
     case underscoreStrong(Index)
+    case tilde(Index)
```
Comment (htinlinn) [inline Marker/Classes/Markdown/EmphasisTag.swift]: call this `tildeStrikethrough` to be consistent with the other cases...

File: Marker/Classes/Markdown/EmphasisTag.swift [hunk 3]
```diff
@@ -20,13 +20,15 @@ internal enum EmphasisTag {
...
     
     /**
      *  Emphasis tag constants.
      */
     struct Constants {
         static let emTagLength = 1
         static let strongTagLength = 2
+        static let tildeLength = 1
```
Comment (htinlinn) [inline Marker/Classes/Markdown/EmphasisTag.swift]: also, here. we should call this `strikethroughTag` length.

File: Marker/Classes/Markdown/MarkdownTag.swift [hunk 4]
```diff
@@ -18,6 +18,7 @@ internal enum MarkdownTag {
     
     case em(Range<Index>)
     case strong(Range<Index>)
+    case strikeThough(Range<Index>)
```
Comment (htinlinn) [inline Marker/Classes/Markdown/MarkdownTag.swift]: just nitpicking here. `strikethrough` is one word, not two according to the dictionary. can you do a find-and-replace, please?

**Review by htinlinn — CHANGES_REQUESTED** [review]

File: Marker/Classes/Markdown/EmphasisTag.swift (hunk 2 above)
Comment (jgsamudio) [reply Marker/Classes/Markdown/EmphasisTag.swift]: 👍

File: Marker/Classes/Markdown/EmphasisTag.swift (hunk 3 above)
Comment (jgsamudio) [reply Marker/Classes/Markdown/EmphasisTag.swift]: 👍

File: Marker/Classes/Markdown/MarkdownTag.swift (hunk 4 above)
Comment (jgsamudio) [reply Marker/Classes/Markdown/MarkdownTag.swift]: 👍

File: Marker/Classes/TextStyle.swift (hunk 1 above)
Comment (jgsamudio) [reply Marker/Classes/TextStyle.swift]: 👍

**Review by htinlinn — APPROVED** [review]

**Pull request merged**